tests/conftest.py

```diff
@@ -36,6 +46,7 @@ def __init__(self, testbed_file):
 def pytest_addoption(parser):
     parser.addoption("--testbed", action="store", default=None, help="testbed name")
     parser.addoption("--testbed_file", action="store", default=None, help="testbed file name")
+    parser.addoption("--disable_loganalyzer", action="store_true", default=False, help="disable loganalyzer analysis for 'loganalyzer' fixture")
```

Comment:
Need comments from community here. Is this useful feature to disable globally from CLI ?
I mean two runs can give different test results depending on this flag.

Reply:
As no more comments from community, keep this options for now.